oioioi/base/permissions.py

```diff
@@ -177,3 +177,7 @@ def not_anonymous(request):
 @make_request_condition
 def is_superuser(request):
     return request.user.is_superuser
+
+@make_request_condition
+def isnt_superuser(request):
+    return not request.user.is_superuser
```

Comment:
`Condition` has `__invert__` implemented, so one can probably just use `~is_superuser`?

Reply:
Yeah, you are right.
I will improve this PR after OI.